### PR TITLE
Create alerts.md

### PR DESCRIPTION
**Pull request by ackerman80**
documenting alerts apis

### CODE REVIEW

**Comment by ackerman80** [comment]
Creating API folder
Creating alerts API documentation

File: docs/apis/alerts.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+# Get All Alerts 
+## Request
+
+- Requires [authentication] (https://github.com/ackerman80/docs/blob/patch-2/docs/apis/authentication.md)
+- HTTP Method: GET
+- URL: http://demo.netsil.io/alerts-service/alerts/
```
Comment (luhkevin) [inline docs/apis/alerts.md]: This link would need to be generic -- demo.netsil.io is just for our demos.

**Comment by luhkevin** [comment]
Just a reminder to squash and merge when you merge this

File: docs/apis/alerts.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
+# Get All Alerts 
+## Request
+
+- Requires [authentication] (https://github.com/ackerman80/docs/blob/patch-2/docs/apis/authentication.md)
```
Comment (jarnoux) [inline docs/apis/alerts.md]: you can probably predict what the url is going to be when this will be merged

File: docs/apis/alerts.md (hunk 1 above)
Comment (jarnoux) [inline docs/apis/alerts.md]: should be just the path + url/path params

File: docs/apis/alerts.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+- HTTP Method: GET
+- URL: http://demo.netsil.io/alerts-service/alerts/
+
+## Response
+Returns an array of all the alerts accessible to the user. Note the *alertId* field which can be used to obtain a specific alert.
```
Comment (jarnoux) [inline docs/apis/alerts.md]: > the *alertId* field

where is that field? Maybe "Note: to retrieve a specific alert, see 'xxx' below"

**Comment by jarnoux** [comment]
To squash your commits, click on the green down-arrow next to "Merge pull request" and choose "Squash and merge". If you can reword your commit messages, choose something informative for this PR.

**Comment by ackerman80** [comment]
First iteration on documenting the alerts and authentication REST calls

**Pull request merged**